### ISSUE TITLE
Rename/Download Files in Upload Date Order

### ISSUE THREAD

**Issue by KHarshawat** [department-of-veterans-affairs/caseflow-efolder#48]
During our usability test we observed most (all?) users requested that the downloaded files be ordered by the date they were uploaded. An important consideration is what we do when we have to skip a file because it can't be downloaded. We could do this in a couple ways...

- Name files to 01_FILENAME.pdf, 02_FILENAME.pdf, etc and skip over numbers when a file is missing. For example if File 03 could not be downloaded we'd go from 02_FILENAME.pdf to 04_FILENAME.pdf
- Name files to UPLOAD-DATE-TIMESTAMP_FILENAME.PDF. The upside is we wouldn't have to do anything tricky with skipping file name numbers but could be more confusing for the user.

We should also add copy indicating how the files will be named when downloaded to the download page.

Me and @shanear will discuss.

**Comment by shanear** [comment]
Additionally, per Clayton, the filenames don't really mean anything. We can potentially name the files based on their file type. Another thing to consider.

**Comment by KHarshawat** [comment]
We decided to `00001.txt`

**Comment by lakohl** [comment]
Looping in @abbyraskin 

- Curious what type of user participated in the usability tests above?

- From what I've observed so far, filenames seem to mean a lot to attorneys working on cases, but less in other use cases e.g. employees processing FOIA requests. 

- Upload date and receipt date don't appear to always be the same thing. (We've observed different users using different types of dates to name/order files.
